services/user-service/index.ts

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import action from "./actions";
+import swaggerJSDoc from 'swagger-jsdoc';
+
+export default {
+  initialize: (options: { database: {username: string, password: string, database: string, host: string, port: number, dialect:  "mysql" | "mariadb" | "postgres"  }, swaggerSpecOption?: swaggerJSDoc.Options }) => {
```

Comment:
can change we change the type of the `database` to `Options` from `sequelize`?

just import it from sequelize and probably put an alias on it

```ts
import { Options as SequelizeOptions } from 'sequelize`;

initialize: (options: { database : SequelizeOptions, swaggerOption?: swaggerJSDoc.Options } }

```